x/X++.xpp

```diff
@@ -0,0 +1,11 @@
+class classHello
+{
+}
+
+static void main(args Args)
+{
+ dialog dialog;
+ dialog = new dialog();
```

Comment:
Is it possible for these two lines to be joined together?

Reply:
No